.github/workflows/run-tests.yml

```
@@ -580,6 +597,23 @@ jobs:
           path: ~/.cache/go-build
           key: go-${{ runner.os }}${{ runner.arch }}-build-${{ env.COMMIT }}
 
+      - name: Get job ID from workflow run
+        id: get_job_id
+        run: |
+          set -eu
```

```suggestion
          set -euo pipefail
```

In case the commands after the pipe fail.